aiida_quantumespresso/parsers/projwfc.py

```diff
@@ -168,16 +171,11 @@ def spin_dependent_subparser(out_info_dict):
         raise QEOutputParsingError('the standard out file does not comply with the official documentation.')
 
     bands_data = BandsData()
-    # Attempts to retrieve the kpoints from the parent calc
-    parent_calc = out_info_dict['parent_calc']
+    kpoints = od['kpoints']
```

Comment:
Huh? Where is this `od` variable coming from? I imagine this is the `out_info_dict` but I can't see where the name change is happening. Anyway, not sure I am a fan of `od`. Not very descriptive name

Reply:
It's here 😄 https://github.com/aiidateam/aiida-quantumespresso/blob/c28d19c370cec6cecf878cb8e93b376899f4d77a/aiida_quantumespresso/parsers/projwfc.py#L130

Reply:
@sphuber I know, the whole code here isn't the most readable, but as I lament in [my comment above](https://github.com/aiidateam/aiida-quantumespresso/pull/747#issuecomment-932989255), I've refactored the code quite a bit in https://github.com/aiidateam/aiida-quantumespresso/pull/749. However, that PR doesn't support all 6.X versions, since I removed some of the logic that deals with very old versions in cleaning up the code. It's also not backwards compatible since I think we shouldn't add all the PDOS files to the repository, since all the info they contain are in the `ProjectionData` output. So I'd leave that for v4.0.0, from which we'll only support QE v6.5+.